swarm/network/kademlia.go

```diff
@@ -206,6 +206,14 @@ func (k *Kademlia) Register(peers []OverlayAddr) error {
 		k.addrCountC <- k.addrs.Size()
 	}
 	// log.Trace(fmt.Sprintf("%x registered %v peers, %v known, total: %v", k.BaseAddr()[:4], size, known, k.addrs.Size()))
+
+	if k.nDepthC != nil {
```

Comment:
Can we please have more comments? What is this check doing? Just helps reading the code faster once one has to look back later (or for new developers)

Reply:
:+1: Will add more comments. There are some on nDepthC and NeighbourhoodDepthC, but they may be far away from this part of the code.